docs/doxygen-user/central_repo.dox

```diff
@@ -94,6 +80,16 @@ Descriptions of the property types:
 - <b>USB Devices</b>
  - USB device properties come from the registry parsing in the Recent Activity Module.
 
+\subsection cr_manage_orgs Manage Organizations
+
+Organizations are stored in the central repository and contain contact information for the given organization. Organizations are used for Hash Databases saved in the central repository, and can also be associated with Autopsy cases. 
```

Comment:
As of now, the Hash Databases panel is still named that, and the list of hashdbs is titled "Hash Databases:" I would also like this standardized, but at the moment it would be inaccurate.

Reply:
Ok